server/src/main/java/org/opensearch/search/sort/FieldSortBuilder.java

```diff
@@ -611,22 +611,32 @@ public static FieldSortBuilder getPrimaryFieldSortOrNull(SearchSourceBuilder sou
      * and configurations return <code>null</code>.
      */
     public static MinAndMax<?> getMinMaxOrNull(QueryShardContext context, FieldSortBuilder sortBuilder) throws IOException {
-        return getMinMaxOrNullInternal(context.getIndexReader(), context, sortBuilder);
+        return getMinMaxOrNullInternal(context.getIndexReader(), context, sortBuilder, null);
```

Comment:
Instead of passing `null`, we can just create the `SortAndFormat` here and pass it. That way the common method will not have any special null handling.

```
public static MinAndMax<?> getMinMaxOrNull(QueryShardContext context, FieldSortBuilder sortBuilder) throws IOException {
          final SortAndFormats sort = SortBuilder.buildSort(Collections.singletonList(sortBuilder), context).get();
          return getMinMaxOrNullInternal(context.getIndexReader(), context, sortBuilder, sort);
}
```

Reply:
Nice ! Thank you for the suggestion.